internal/cmd/config/config.go

```diff
@@ -349,6 +349,8 @@ func Parse(d string) (*Config, error) {
 					}
 				case int:
 					result.Controller.Database.MaxOpenConnections = t
+				default:
+					return nil, fmt.Errorf("Database max open connections value is not an int: %v", t)
```

Comment:
IIRC this message goes all the way back to the user in their CLI. In that context, I think this error might be a bit misleading since we also support strings (i.e `"8"` is a valid value).

Maybe more like "Database max open connections: unsupported type %q", with `%q` being the value of   `reflect.TypeOf(t).String()`, which is the actual type. WDYT?

Reply:
Sounds good to me!